File: eta_calculator.py
class GougleMaps:
    def __init__(self): pass
    
    def calculate_eta(self, distance: int, speed: int) -> float:
        eta = round(distance / speed, 2)
        return eta

def main():
    maps = GougleMaps()
    eta1 = maps.calculate_eta(distance=300, speed=80)
    eta2 = maps.calculate_eta(distance=193, speed=47)
    print(eta1)
    print(eta2)


if __name__ == "__main__":
    main()

File: eta_calculator.c
#include <stdio.h>
#include <string.h>
#include <math.h>

float calculate_eta(int distance, int speed) {
    float eta;
    eta = (float)distance / speed;
    return eta;
}


int main() {
    int distance, speed;
    float eta;

    printf("Enter distance (in miles): ");
    scanf("%d", &distance);

    printf("Enter speed (in mph): ");
    scanf("%d", &speed);

    if (speed == 0) {
        printf("Speed cannot be zero.\n");
        return 1;
    }

    eta = calculate_eta(distance, speed);

    printf("Travel time: %.2f hours\n", eta);

    return 0;
}
